mmdet/ops/corner_pool/src/corner_pool.cpp

```diff
@@ -0,0 +1,270 @@
+#include <torch/torch.h>
```

Comment:
If this file is modified from other repos, add a reference.

Reply:
Although only CPU mode is supported, we may preserve the APIs.

Reply:
> Although only CPU mode is supported, we may preserve the APIs.

Should I do any modification to "preserve the APIs"?